test/push-scheduler/in/.keboola/manifest.json

```diff
@@ -0,0 +1,43 @@
+{
+  "version": 1,
+  "project": {
+    "id": %%TEST_KBC_PROJECT_ID%%,
+    "apiHost": "%%TEST_KBC_STORAGE_API_HOST%%"
+  },
+  "sortBy": "path",
+  "naming": {
+    "branch": "{branch_name}",
+    "config": "{component_type}/{component_id}/{config_name}",
+    "configRow": "rows/{config_row_name}",
+    "sharedCodeConfig": "_shared/{target_component_id}",
+    "sharedCodeConfigRow": "codes/{config_row_name}",
+    "variablesConfig": "variables",
+    "variablesValuesRow": "values/{config_row_name}"
+  },
+  "allowedBranches": [
+    "__all__"
+  ],
+  "ignoredComponents": [],
+  "branches": [
+    {
+      "id": %%TEST_BRANCH_MAIN_ID%%,
+      "path": "main"
+    }
+  ],
+  "configurations": [
+    {
+      "branchId": %%TEST_BRANCH_MAIN_ID%%,
+      "componentId": "ex-generic-v2",
+      "id": "22222",
```

Comment:
Drobnost na zvatenie, ... zvoli by som skor ID `1/2` alebo `123/456` ... ak ten test niekomu spadne, tak sa to bud jednoduchsie debugovat ... ked sme riesili ten problem, tak som asi na 3x pozeral kde je `22222` a kde `22223` :) 

Mozno ze si to skopiroval z nejakeho testu od TK :) on mal take cisla rad :)

Reply:
Nemusis to menit, skor do buducna ....